test/e2e/testdata/images/tmp-writer/writer.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+
+touch "/tmp/foo.txt"
```

Comment:
this test is going to pass even if creation of the file is not possible, because the script does not configure to exit immediatelly

Reply:
thanks for the catch! updated the script

Reply:
lines 3-6 could be shortened to:

```
touch "/tmp/foo.txt" || exit 1
```

Reply:
done